### DISCUSSION TITLE
LinkFieldValue and ImageFieldValue exports missing

### DISCUSSION THREAD

**Discussion by eclipticrick** [Sitecore/jss#1677] [Help] [answered]
### Description

The current `LinkFieldValue` and `ImageFieldValue` are only exported from the deprecated `@sitecore-jss/sitecore-jss-manifest` package, not yet from `@sitecore-jss/sitecore-jss-angular`. can these exports be added here as well?

### Additional information

currently migrating from v19 to v21

**Comment by art-alexeyenko** [comment, marked as answer]
Thanks for reporting this! I've registered an item in our backlog to add those exports.

**Reply by art-alexeyenko** [reply]
The change is introduced in v21.6 and up
